<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="8" total="9" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Parallel test suite" duration-ms="22412" started-at="2018-03-16T12:19:51Z" finished-at="2018-03-16T12:20:13Z">
    <groups>
    </groups>
    <test name="Regression 1" duration-ms="22412" started-at="2018-03-16T12:19:51Z" finished-at="2018-03-16T12:20:13Z">
      <class name="Com.GeneoStudent.Testcases.CoverPageTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:Com.GeneoStudent.Testcases.CoverPageTest@5f341870]" name="setup" is-config="true" duration-ms="18179" started-at="2018-03-16T17:49:51Z" finished-at="2018-03-16T17:50:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="Cover_Page_menu_Change_Password_Button_Test()[pri:10, instance:Com.GeneoStudent.Testcases.CoverPageTest@5f341870]" name="Cover_Page_menu_Change_Password_Button_Test" duration-ms="4206" started-at="2018-03-16T17:50:09Z" finished-at="2018-03-16T17:50:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Change Password window Failed to open expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Change Password window Failed to open expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertFalse(Assert.java:62)
	at Com.GeneoStudent.Testcases.CoverPageTest.Cover_Page_menu_Change_Password_Button_Test(CoverPageTest.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Cover_Page_menu_Change_Password_Button_Test -->
      </class> <!-- Com.GeneoStudent.Testcases.CoverPageTest -->
    </test> <!-- Regression 1 -->
  </suite> <!-- Parallel test suite -->
</testng-results>
